dotCMS/src/integration-test/java/com/dotcms/util/TestInitialContext.java

```diff
@@ -72,8 +73,8 @@ private void loadProperties() throws NamingException {
 
         try (InputStream resourceStream = loader.getResourceAsStream("db-config.properties")) {
             prop.load(resourceStream);
-        } catch (IOException e) {
-            throw new NamingException("Unable to get properties from db-config.properties");
+        } catch (Exception e) {
+            Logger.info(this.getClass(), "Unable to get properties from file db-config.properties - let's try the environment");
```

Comment:
Yeah, wondering if .error or .warning would be better, or if it would be too much. Thinking of the case of no db file and no env properties. Maybe info is fine